src/server/tran_server.cpp

```diff
@@ -457,15 +457,31 @@ tran_server::connection_handler::set_connection (cubcomm::channel &&chn)
   // TODO: to reduce contention as much as possible, should be equal to the maximum number
   // of active transactions that the system allows (PRM_ID_CSS_MAX_CLIENTS) + 1
 
-  cubcomm::send_queue_error_handler no_transaction_handler { nullptr };
-  // Transaction server will use message specific error handlers.
-  // Implementation will assert that an error handler is present if needed.
+  cubcomm::send_queue_error_handler default_error_handler = [this]
+      (css_error_code error_code, bool &abort_further_processing)
+  {
+    // Remove the connection_handler if the internal socket is closed. It's been disconnected abnormally.
```

Comment:
Nit. Just to make it explicit.

```suggestion
    abort_further_processing = false;
    // Remove the connection_handler if the internal socket is closed. It's been disconnected abnormally.
```

Reply:
done